install/kubernetes/Makefile

```diff
@@ -48,7 +49,7 @@ update-chart: $(ROOT_DIR)/VERSION # Update chart versions to point to the curren
 		| xargs -L 1 $(SED) -i -e 's/'$(CILIUM_CHART_REGEX)'/\1 '$(VERSION)'/g'
 	$(QUIET)$(SED) -i 's;icon:.*;icon: $(LOGO_BASE_URL)/cilium@$(CILIUM_BRANCH)/$(LOGO_PATH);' $(CHART_FILE)
 
-check-values-yaml:
+check-values-yaml: cilium/values.schema.json
```

Comment:
This target creates a dependency on generating the schema, but who uses the schema to validate the values?

Reply:
:+1: looks like it's supposed to be picked up by `helm lint` too, which we already run as part of the CI.